code/modules/jobs/job_types/roguetown/garrison/mayor.dm

```diff
@@ -65,4 +64,32 @@
 		H.change_stat("speed", -3)
 		H.change_stat("intelligence", 5)
 		ADD_TRAIT(H, TRAIT_WANTED_POSTER_READ, TRAIT_GENERIC)
+		ADD_TRAIT(H, TRAIT_BOGVULNERABLE, TRAIT_GENERIC)
+		ADD_TRAIT(H, TRAIT_SEEPRICES_SHITTY, TRAIT_GENERIC)
+
+	H.verbs |= list(/mob/living/carbon/human/proc/request_outlaw_mayor)
+
+/mob/living/carbon/human/proc/request_outlaw_mayor()
+	set name = "Request Outlaw"
+	set category = "Martial Law"
```

Comment:
Rename this category to something else, Martial law doesn't make much sense for for the mayor.
"Juris Civilis", Perhaps?
Anything works as long as it's fitting for the role.